source/assets/css/components/_doc-navigation.scss

```diff
@@ -0,0 +1,16 @@
+.sl-l-medium-holy-grail__body {
```

Comment:
probably don't need the nesting here since it's only one context inside. So maybe instead:

```scss
.sl-l-medium-holy-grail__body.sl-js-nav--is-sticky  .sl-c-list-navigation-wrapper {
  position: sticky;
  top: 0;
  bottom: 0;
  overflow-y: scroll;

  @include sl-breakpoint--large { 
    height: 100vh;
  }
}
```